### ISSUE TITLE
Images missing alt text

### ISSUE THREAD

**Issue by augustina-jpg** [augustina-jpg/scoopdope#1047]
Several course thumbnail images lack `alt` text. Add descriptive alt text to all images.

**Comment by janetobande** [comment]
@janetobande has applied to work on this issue as part of the **Stellar Wave Program**'s 9th wave.



> Hello, i would like to work on this issue, i have attended to something similar to this issue and i did well 
> Pls give me a chance to work on this too


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/1329c87f-70c6-4e0a-a59e-940280592c35) or assign @janetobande to this issue.

**Comment by griffin-de-dev** [comment]
@griffin-de-dev has applied to work on this issue as part of the **Stellar Wave Program**'s 9th wave.



> I would like to work on this issue 


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/1329c87f-70c6-4e0a-a59e-940280592c35) or assign @griffin-de-dev to this issue.
